mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -0,0 +1,274 @@
+
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as cols
+
+import numpy as np
+import xarray as xr
+import datetime
+from netCDF4 import Dataset as netcdf_dataset
+
+from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
+
+from ..shared.plot.plotting import plot_global_comparison
+
+from ..shared.interpolation.interpolate import interp_fields, init_tree
+
+def mld_modelvsobs(config):
+    
+    indir = config.get('paths','archive_dir_ocn')
+    plots_dir = config.get('paths','plots_dir')
+    obsdir = config.get('paths','obs_mlddir')
+    
+    casename = config.get('case','casename')
+    meshfile = config.get('data','mpas_meshfile')
+
+    climo_yr1 = config.getint('time','climo_yr1')
+    climo_yr2 = config.getint('time','climo_yr2')
+    yr_offset = config.getint('time','yr_offset')
+
+    infiles = "".join([indir,"/am.mpas-o.timeSeriesStats.????-*.nc"])
+
+    monthdictionary={'Jan':1,'Feb':2,'Mar':3,'Apr':4,'May':5,'Jun':6,'Jul':7,'Aug':8,'Sep':9,'Oct':10,
+                         'Nov':11,'Dec':12,'JFM':np.array([1,2,3]),'AMJ':np.array([4,5,6]),'JAS':np.array([7,8,9]),
+                         'OND':np.array([10,11,12]),'ANN':np.arange(1,13)}
+                         
+    outputTimes = config.getlist('mld_modelvsobs','comparisonTimes')
+
+    obs_filename = "%s/holtetalley_mld_climatology.nc" % obsdir
+    
+      # Load in model data
+    ds = xr.open_mfdataset(infiles, preprocess=lambda x: preprocess_mpas(x, yearoffset=yr_offset,
+                                timeSeriesStats=True,
```

Comment:
Same issue with indentation as above (`timeSeriesStats` should be left justified with `inflies`